services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/diagnostic_reports/diagnostics/diagnostic_activity_packs.tsx

```diff
@@ -100,11 +78,28 @@ const PostSection = ({ post, activityId, unitTemplateId, name, }) => {
 }
 
 const Diagnostic = ({ diagnostic, }) => {
+  const [skillsGrowth, setSkillsGrowth] = React.useState(null)
   const { name, pre, post, } = diagnostic
+
+  React.useEffect(() => {
+    if (post && post.assigned_count) {
+      getSkillsGrowth()
+    }
+  }, [])
+
+  function getSkillsGrowth() {
+
+    requestGet(`/teachers/progress_reports/skills_growth/${pre.classroom_id}/post_test_activity_id/${post.activity_id}/pre_test_activity_id/${pre.activity_id}`,
+      (data) => {
+        setSkillsGrowth(data.skills_growth)
+      }
+    )
+  }
+
   let postAndGrowth = <PostInProgress name={name} />
   if (pre.post_test_id) {
     const growthSummaryLink = resultsLink(true, pre.post_test_id, pre.classroom_id, pre.unit_id)
-    postAndGrowth = post.assigned_count ? <React.Fragment><PostSection post={post} /><GrowthSummary growthSummaryLink={growthSummaryLink} showGrowthSummary={true} skillsGrowth={post.skills_count - pre.skills_count} /></React.Fragment> : <React.Fragment><PostSection activityId={pre.post_test_id} name={name} unitTemplateId={post.unit_template_id} /><GrowthSummary name={name} /></React.Fragment>
+    postAndGrowth = post.assigned_count ? <React.Fragment><PostSection post={post} /><GrowthSummary growthSummaryLink={growthSummaryLink} showGrowthSummary={true} skillsGrowth={skillsGrowth} /></React.Fragment> : <React.Fragment><PostSection activityId={pre.post_test_id} name={name} unitTemplateId={post.unit_template_id} /><GrowthSummary name={name} /></React.Fragment>
```

Comment:
I think it'd be easier to read if we broke this out onto multiple lines since it's pretty long right now